ipc/provider/src/checkpoint.rs

```diff
@@ -106,18 +106,28 @@ impl<T: BottomUpCheckpointRelayer + Send + Sync + 'static> BottomUpCheckpointMan
         log::info!("launching {self} for {submitter}");
 
         loop {
-            if let Err(e) = self.submit_checkpoint(&submitter).await {
-                log::error!("cannot submit checkpoint for submitter: {submitter} due to {e}");
-            }
-
+            self.submit_checkpoint(&submitter).await;
             tokio::time::sleep(submission_interval).await;
         }
     }
 
     /// Submit the checkpoint from the target submitter address
-    pub async fn submit_checkpoint(&self, submitter: &Address) -> Result<()> {
-        self.submit_last_epoch(submitter).await?;
-        self.submit_next_epoch(submitter).await
+    pub async fn submit_checkpoint(&self, submitter: &Address) {
+        let next_submission_height = if let Ok(h) = self.next_submission_height().await {
+            h
+        } else {
+            log::error!("cannot fetch next submission height for submitter {submitter}");
```

Comment:
```suggestion
            log::error!(submitter, "cannot fetch next submission height");
```

Would it be worth logging the actual error as well?